### ISSUE TITLE
key images for bee species for wikidata with commons license

### ISSUE THREAD

**Issue by jhpoelen** [Big-Bee-Network/Big-Bee-community-discussion#8]
Hey @seltmann et al. ,

@Daniel-Mietchen was looking for a corpus of bee images that can be used as depiction of bee species listed on wikidata. He claims that many bee species known to Wikidata do not yet have a pictures associated to it. 

@Daniel-Mietchen - can you provide a list of bee species known to wiki data and whether or not they have an image.

@seltmann do you think it'd be worthwhile to re-use big-bee images for use in Wikidata?

Note that GloBI already uses wikidata as a way to pull up thumbnail to decorate taoxnomic name names in web result pages.

See attached captured from  https://www.globalbioticinteractions.org/?interactionType=interactsWith&sourceTaxon=Apis  

![Screenshot from 2022-06-27 16-29-34](https://user-images.githubusercontent.com/1084872/176038767-d526280c-3f10-403e-b004-2bce18f349c5.png)

**Comment by seltmann** [comment]
@jhpoelen @Daniel-Mietchen yes! I would be excited to help provide bee images via Wikidata. Our images are under CC0 license as I know this is an important step in providing access. Let me know what other steps are required.

**Comment by Daniel-Mietchen** [comment]
@seltmann @jhpoelen Great that the images are CC0. Wikidata currently knows of about 4.5k bee species, about 10% of which have an image: https://w.wiki/5Mzw .

For those that do have an image, here is a gallery query: https://w.wiki/5M$4 and a snapshot:

![Screenshot from 2022-06-28 17-01-48](https://user-images.githubusercontent.com/465923/176213428-c3808d19-e737-4bc5-9a19-bccb4f2f7121.png)

In order to show up in Wikidata, the images would first have to be imported to Wikimedia Commons, for which there are various options that would require an API or a dump of your images and the associated metadata. Do you have something along those lines?
